samples/CMakeLists.txt

```diff
@@ -63,3 +63,20 @@ if (CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE)
     SOURCE_DIR ${NRF_DIR}/samples/nrf5340/empty_app_core
     DOMAIN nrf5340pdk_nrf5340_cpuapp)
 endif()
+
+if (CONFIG_OT_RCP_NRF53)
```

Comment:
It looks that modification in this file is not needed. Building appropriate image for the Network core shall be provided by 802.15.4 serialization, when such serialization is enabled in the Application core. I don't think it is connected with OpenThread RCP.
When we add support for 802.15.4 in nRF53 Net core and serialization, we'll append similar modifications to this file. At the moment I would remove it.
Or am I missing something?